resources/views/users/dashboard.blade.php

```diff
@@ -24,5 +31,25 @@
             </div>
 
         </div>
-    </section><!-- End Services Section -->
+    </section>
+    <script src="https://cdn.jsdelivr.net/npm/sweetalert2@11"></script>
```

Comment:
ads script in @push in frontend authenticate layouts,
suppose frontend authenticate layouts have js and  css
@stack('js');